### ISSUE TITLE
Helm Chart 1.9.25 fails to render

### ISSUE THREAD

**Issue by fculpo** [grafana/oncall#5065]
### What went wrong?

**What happened**:
- Upgrade post 1.9.23 with default value (or custom) fails with
- This seems to be a "conflict" between sub charts

```
❯ helm template grafana/oncall
Error: template: oncall/charts/redis/templates/replicas/serviceaccount.yaml:13:14: executing "oncall/charts/redis/templates/replicas/serviceaccount.yaml" at <include "common.labels.standard" (dict "customLabels" .Values.commonLabels "context" $)>: error calling include: template: oncall/charts/mariadb/charts/common/templates/_labels.tpl:6:27: executing "common.labels.standard" at <include "common.names.name" .>: error calling include: template: oncall/charts/mariadb/charts/common/templates/_names.tpl:6:18: executing "common.names.name" at <.Chart.Name>: nil pointer evaluating interface {}.Name
```

**What did you expect to happen**:
- upgrade should succeed
- chart should render with default values

### How do we reproduce it?

1. helm template grafana/oncall (1.9.25)

### Grafana OnCall Version

v1.9.25

### Product Area

Helm/Kubernetes/Docker

### Grafana OnCall Platform?

Kubernetes

**Comment by joeyorlando** [comment]
this should get fixed in v1.9.29 of the chart (via https://github.com/grafana/oncall/pull/5069; on an unrelated note we had a GitHub Actions workflow hiccuping publishing > v1.9.25 of the helm chart, that has also been fixed)

**Comment by joeyorlando** [comment]
FYI, v1.9.29 chart has now been released
